README.md

```diff
@@ -55,16 +55,16 @@ Let me know! Open an issue (or PR) or write in [slack](https://elmlang.slack.com
 
 ### Setup
 
-```elm
-elm package install
+```shell
+elm-package install
```

Comment:
What's wrong with `elm package` instead of `elm-package`? They do exactly the same on my system. (just curious)